pkgs/games/planetaryannihilation/default.nix

```diff
@@ -16,7 +16,8 @@ assert ((config.planetary_annihilation or null).sha256 or null) != null;
 */
 
 stdenv.mkDerivation {
-  name = "planetary-annihalation";
+  pname = "planetary-annihalation";
+  version = "62857";
 
   src = fetchurl {
```

Comment:
We can use the following and drop the `config` stuff completely.


```
  src = requireFile {
    message = "This file has to be downloaded manually.";
    name = "PA_Linux_${version}.tar.bz2";
    sha256 = "0imi3k5144dsn3ka9khx3dj76klkw46ga7m6rddqjk4yslwabh3k";
  };
```

Reply:
should be fixed